src/models/comments.js

```diff
@@ -0,0 +1,13 @@
+export default (sequelize, DataTypes) => {
+  const Comments = sequelize.define('Comments', {
+    userId: DataTypes.INTEGER,
```

Comment:
@erickyvand may you update the comment model so that it caters for other types of comments